src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+    private int lengthEnumColor = Color.values().length;
```

Comment:
I don't think it is necessary to put this color length as class level field, it is already a constant by itself